tests/codegen-units/item-collection/cross-crate-closures.rs

```diff
@@ -3,14 +3,13 @@
 //@ compile-flags:-Zprint-mono-items=eager -Zinline-mir=no
 
 #![deny(dead_code)]
-#![feature(start)]
+#![crate_type = "lib"]
 
 //@ aux-build:cgu_extern_closures.rs
 extern crate cgu_extern_closures;
 
-//~ MONO_ITEM fn start @@ cross_crate_closures-cgu.0[Internal]
-#[start]
-fn start(_: isize, _: *const *const u8) -> isize {
+//~ MONO_ITEM fn start @@ cross_crate_closures-cgu.0[External]
+pub fn start(_: isize, _: *const *const u8) -> isize {
```

Comment:
Maybe add `#[no_mangle]` to ensure that the function is getting codegened locally even if we in the future make cross-crate inlining apply in much more cases?